accounts/main.go

```diff
@@ -33,14 +35,20 @@ func main() {
 
 	logger := log.New(os.Stdout, "server: ", log.Flags())
 
+	err := godotenv.Load("./.env")
+	if err != nil {
+		panic(err)
+	}
+	log.Println(strings.Split(os.Getenv("CORS"), ","))
+
 	r := chi.NewRouter()
 	{
 		r.Use(middleware.RequestID)
 		r.Use(middleware.RealIP)
 		r.Use(middleware.Logger)
 		r.Use(middleware.Recoverer)
 		r.Use(cors.Handler(cors.Options{
-			AllowedOrigins:   []string{"https://*", "http://*"},
+			AllowedOrigins:   strings.Split(os.Getenv("CORS"), ","),
```

Comment:
`CORS` env name is a bit ambiguous, AllowedOrigins would be a better name but it's not the end of the world